smoke-tests/otel-collector-contrib-windows/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2019
+ADD https://github.com/signalfx/splunk-otel-collector/releases/latest/download/otelcol_windows_amd64.exe otelcol_windows_amd64.exe
```

Comment:
I think we need contrib here, no?

Reply:
🤦  I forgot about the most important thing